sonaranalyzer-dotnet/src/SonarAnalyzer.VisualBasic/Rules/ConditionalStructureSameImplementation.cs

```diff
@@ -116,5 +128,10 @@ protected override void Initialize(SonarAnalysisContext context)
             context.ReportDiagnostic(Diagnostic.Create(rule,
                 location, locationProvider.First().GetLineNumberToReport(), constructType));
         }
+
+        private static bool IsNotIgnored(StatementSyntax statement)

```

Comment:
Can this be positive? For example rename to IsIgnored or IsIgnoredStatement and then negate at the point of calling.